lms/djangoapps/courseware/tests/test_entrance_exam.py

```diff
@@ -283,13 +283,15 @@ def test_entrance_exam_score(self):
         """
         test entrance exam score. we will hit the method get_entrance_exam_score to verify exam score.
         """
-        exam_score = get_entrance_exam_score(self.request, self.course)
+        with self.assertNumQueries(1):
+            exam_score = get_entrance_exam_score(self.request, self.course)
```

Comment:
N.B. This number is only valid in small courses. As the course gets larger, then this will actually issue multiple queries (because you're using the `chunked_filter`.